src/main/scala/server/Main.scala

```
@@ -0,0 +1,166 @@
+
+import scala.collection.mutable.HashMap
+
+import akka.actor.typed.{ActorRef, ActorSystem}
+import soc.akka.messages.StateMessage
+import soc.game.inventory.Inventory.{NoInfo, PerfectInfo}
+
+import soc.akka.{GameBehavior, PlayerBehavior, MoveSaverBehavior, MoveResultProvider, RandomMoveResultProvider}
+import akka.actor.PoisonPill
+import akka.actor.typed.{ActorRef, ActorSystem}
+import soc.akka.messages.{GameMessage, Terminate}
+import soc.game.dice.NormalDice
+import soc.game.player.moveSelector.PossibleMoveSelector
+import soc.game._
+import soc.game.board.{BaseBoardConfiguration, BaseCatanBoard}
+import soc.game.inventory.Inventory.{NoInfo, PerfectInfo}
+import soc.game.inventory._
+import soc.storage.aws.AWSMoveSaver
+
+import scala.util.Random
+import scala.concurrent.ExecutionContext.Implicits.global
+import soc.aws.client.CatanGameStoreClientFactory
+import soc.game.inventory.resources.CatanResourceSet
+import soc.game.inventory.InventoryManager._
+import soc.simulation.SimulationQueue
+
+import soc.game.{GameConfiguration, GameRules}
+import scala.util.{Success, Failure}
+
+object GameContext {
+  implicit val random = new Random()
+
+  val boardConfig = BaseCatanBoard.randomBoard
+
+  val dice = NormalDice()
+
+ 
+  implicit val gameRules = GameRules()
+
+  val dCardDeck: List[DevelopmentCard] = DevelopmentCardDeckBuilder.buildDeckByCardTypeAndAmount(
+    Map(Knight -> Knight.initAmount,
+      CatanPoint -> CatanPoint.initAmount,
+      RoadBuilder -> RoadBuilder.initAmount,
+      Monopoly -> Monopoly.initAmount,
+      YearOfPlenty -> YearOfPlenty.initAmount)
+  )
+
+  import CatanResourceSet._
+  import io.circe.generic.auto._
+  val awsGameSaver = new AWSMoveSaver[BaseBoardConfiguration](CatanGameStoreClientFactory.createClient())
+  val moveSaverActor: ActorRef[GameMessage] = ActorSystem(MoveSaverBehavior.moveSaverBehavior(awsGameSaver), "moveSaver")
+
+  val randSelector = PossibleMoveSelector.randSelector[NoInfo]
+
+  val players = Map(
```

Pull this out so clients can add their own players. Only fall back on a random player as default.